pkg/platform/version.go

```diff
@@ -50,7 +50,7 @@ func LatestCompatibleVersion(ctx context.Context) (string, error) {
 		return MinimumVersionTag, nil
```

Comment:
Can you change the actual min version too, in pkg/platform/version.go

Reply:
yes, it's done as per our slack discussion.